src/ddqa/utils/jira.py

```diff
@@ -194,6 +196,25 @@ async def update_issue_status(self, client: ResponsiveNetworkClient, issue: Jira
 
         return new_issue
 
+    def get_github_user_id_from_jira_user_id(self, jira_user_id: str) -> str:
+        for github_id, jira_id in self.config.members.items():
+            if jira_id == jira_user_id:
+                return github_id
```

Comment:
A little idea for an optimization: cache the reverse mapping to avoid linear search for every lookup. Moving the member mapping to its own little class could help do that.

Reply:
I extracted that part to the config itself and added a cache that has one caveat, but it should be fine for this application